hs_core/tests/api/native/test_add_resource_files.py

```diff
@@ -81,3 +82,4 @@ def test_add_files(self):
         self.assertTrue(self.n1 in file_list, "file 1 has not been added")
         self.assertTrue(self.n2 in file_list, "file 2 has not been added")
         self.assertTrue(self.n3 in file_list, "file 3 has not been added")
+        delete_resource(res.short_id)
```

Comment:
Why can't we use everywhere where resource needs to be deleted for cleanup  something like this?
```BaseResource.objects.all().delete()```  # where all objects needs to be deleted
OR
```self.specific_resource.delete()``` # where only a specific resource needs to be deleted

Then the changes made in ```delete_resource``` won't be necessary.